plugin-testing-framework/definitions/replace.d.ts

```diff
@@ -0,0 +1,13 @@
+interface ReplaceOptions {
+	regex: string;
+	replacement: string;
+	paths: Array<string>;
+	recursive: boolean;
+	silent: boolean;
```

Comment:
I agree it's not ideal, and yes we could, you could always submit it there if you want, otherwise I think it's fine.